qt/applications/workbench/workbench/plugins/editor.py

```diff
@@ -73,6 +76,22 @@ def app_closing(self):
         """
         return self.editors.close_all()
 
+    def dragEnterEvent(self, event):
+        data = event.mimeData()
+        if data.hasText() and data.hasUrls:
+            filepaths = [url.toLocalFile() for url in data.urls()]
+            for filepath in filepaths:
+                if filepath.endswith('.py') or filepath.endswith('.pyw'):
```

Comment:
Can we define the accepted extensions as a class-level tuple to avoid repeating them?